src/Container/ApplicationFactory.php

```diff
@@ -144,6 +145,24 @@ public function __invoke(ContainerInterface $container)
     }
 
     /**
+     * Sets provided PHP settings, if any.
+     *
+     * @param ContainerInterface $container
+     */
+    private function setPhpSettings(ContainerInterface $container)
+    {
+        $config = $container->has('config') ? $container->get('config') : [];
+
+        if (!isset($config['php_settings']) || !is_array($config['php_settings'])) {
```

Comment:
if the value is not an array I guess an exception should be raised instead silently ignore.